### PR TITLE
net-im/bitlbee: enable py3.10, disable py3.7 and EAPI bump in live ebuild

### PR DESCRIPTION
**Pull request by arkamar**
- enable py3.10 and disable py3.7 in both  `3.6-r1` and live ebuild
- bump to EAPI 8 in live ebuild

### CODE REVIEW

File: net-im/bitlbee/bitlbee-3.6-r1.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (arthurzam) [inline net-im/bitlbee/bitlbee-3.6-r1.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```
`python3_7` is nop for some time, so we can just remove it.

File: net-im/bitlbee/bitlbee-9999.ebuild [hunk 2]
```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2022 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=7
+EAPI=8
 
-PYTHON_COMPAT=( python3_{7..9} )
+PYTHON_COMPAT=( python3_{7..10} )
```
Comment (arthurzam) [inline net-im/bitlbee/bitlbee-9999.ebuild:6]:
```suggestion
PYTHON_COMPAT=( python3_{8..10} )
```
`python3_7` is nop for some time, so we can just remove it.

**Review by arthurzam — CHANGES_REQUESTED** [review]
Great job, just very small nit changes.

File: net-im/bitlbee/bitlbee-9999.ebuild (hunk 2 above)
Comment (arkamar) [reply net-im/bitlbee/bitlbee-9999.ebuild]: done, thnaks for the review.

**Review by arthurzam — APPROVED** [review]
Great job! Thank you for the contribution.

**Pull request closed without merging**